fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisLexer.g4

```diff
@@ -118,6 +118,7 @@ AUTO_INCREMENT: 'AUTO_INCREMENT';
 BACKEND: 'BACKEND';
 BACKENDS: 'BACKENDS';
 BACKUP: 'BACKUP';
+BATCH: 'BATCH';
```

Comment:
add `BATCH` keyword to `nonReserved` in DorisParser.g4

Reply:
https://github.com/apache/doris/pull/36606/files#diff-19b4b34cfa5ffde0dc8660d0df94d019a848eafad24f80821e300fa62947c185R1045 
I think this has already been added.